main.go

```diff
@@ -161,6 +153,31 @@ func scytale(arguments []string) int {
 	return 0
 }
 
+func loadTracing(v *viper.Viper, appName string) (candlelight.Tracing, error) {
+	var tracing = candlelight.Tracing{
+		Enabled:        false,
+		Propagator:     propagation.TraceContext{},
+		TracerProvider: trace.NewNoopTracerProvider(),
+	}
+	if v.IsSet(tracingConfigKey) {
```

Comment:
You can probably check if `tracingConfigKey` is not set and if so, return then? It would help keep things more flat.

Reply:
I was using the outdated code. Unmarshalling straight into the struct works too.